.changeset/brown-wasps-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/build-utils": patch
```

Comment:
I think it's debatable if this is a versioned change, but I think it counts because the interface (to a dev) changes and there could always be a typo that breaks syntax.